src/controllers/cgnController.ts

```diff
@@ -15,9 +15,10 @@ import {
   IResponseSuccessRedirectToResource
 } from "italia-ts-commons/lib/responses";
 
-import { CgnStatus } from "generated/io-cgn-api/CgnStatus";
+import { CgnStatus } from "generated/cgn/CgnStatus";
 import CgnService from "src/services/cgnService";
-import { InstanceId } from "generated/io-cgn-api/InstanceId";
+import { InstanceId } from "generated/cgn/InstanceId";
+import { CgnActivationDetail } from "generated/cgn/CgnActivationDetail";
```

Comment:
Something strange with these imports. Why don't use relative path to related files?
es: 
```
import { CgnStatus } from "../../generated/cgn/CgnStatus";
```